src/utils/api/tracker.ts

```diff
@@ -11,11 +11,11 @@ const getTrackerRegistry = async () => {
 
   if (tracker.source === 'http') {
     return {
-      getAllTrackers: () => [{
-        http: tracker.http,
-      }],
+      getAllTrackers: () => tracker.http.map((url) => ({
+        http: url,
+      })),
       getTracker: () => ({
-        http: tracker.http,
+        http: tracker.http[0],
```

Comment:
@hpihkala This logic needs some clarification, how do we select the correct tracker for the stream?

Reply:
@juhah is this still an open question, or we resolved it in the call?

Reply:
@hpihkala now resolved, switched to using `Utils.createTrackerRegistry` from `streamr-client-protocol`. Testnet 1 config is now:

```js
[{
    id: '0x49D45c17bCA1Caf692001D21c38aDECCB4c08504',
    http: 'https://testnet1.streamr.network:30300',
    ws: 'wss://testnet1.streamr.network:30300',
}]
```

Previously `id` and `ws` were not defined but added for to match `testnet2` config format... Those should be the correct values?